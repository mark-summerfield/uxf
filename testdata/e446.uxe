uxf 1
[thing]
