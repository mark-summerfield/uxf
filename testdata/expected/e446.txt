uxf:E446:e446.uxe:1:expected list vtype, got thing
